packages/scss/src/components/chip/component.scss

```diff
@@ -17,37 +18,56 @@
 
 	@at-root ($atRoot) {
 		.chip-kill {
-			background-color: var(--palettes-text, var(--palettes-grey-700));
-			color: var(--palettes-700, var(--colors-white-color));
-			border: 0;
-			border-radius: 50%;
-			height: 0.75rem;
-			width: 0.75rem;
-			transition-duration: var(--commons-animations-durations-fast);
-			transition-property: background-color;
-			padding: 0;
-			overflow: hidden;
+			@include reset.button;
+			display: grid;
+			grid-template-columns: 1fr;
+			grid-template-rows: 1fr;
+			grid-template-areas: 'main';
+			align-items: normal;
+			width: var(--components-chip-kill-size);
+			height: var(--components-chip-kill-size);
+			border-radius: 50% !important;
 			cursor: pointer;
 
+			&::before,
+			&::after {
+				content: '';
+				grid-area: main;
+			}
+
 			&::before {
-				@include icon.generate('sign_close');
-				display: flex;
-				justify-content: center;
-				align-items: center;
-				line-height: 0.75rem;
-				font-size: 0.75rem;
+				width: 75%;
+				height: 75%;
+				border-radius: 50%;
+				place-self: center;
+				background-color: var(
+					--components-chip-kill-cross-color,
+					var(--palettes-700, var(--colors-white-color))
+				);
+			}
+
+			&::after {
+				mask-image: var(--components-chip-kill-background-image);
+				mask-size: var(--components-chip-kill-size);
+				mask-repeat: no-repeat;
```

Comment:
@BertrandPodevin I don't get why it has to be that complicated. This solution requires a grid, 2 pseudo elements, a mask and inline svg... but could be basically be done using just a single inline svg centered in a flex box.
Am I missing a point?